examples/static_react_task_with_tips/hydra_configs/conf/example.yaml

```diff
@@ -7,6 +7,7 @@ mephisto:
   blueprint:
     task_source: ${task_dir}/webapp/build/bundle.js
     link_task_source: true
+    tips_location: ${task_dir}/outputs/tips.csv
```

Comment:
Yeah, I am adding it to the BlueprintArgs class instead to avoid the need to add the tips_location property itself to each blueprint.

This is causing a github actions testing error that I have to solve though.